ntcore/src/test/native/cpp/ConnectionListenerTest.cpp

```diff
@@ -23,18 +23,16 @@ class ConnectionListenerTest : public ::testing::Test {
     nt::DestroyInstance(client_inst);
   }
 
-  void Connect(const char* address, unsigned int port3, unsigned int port4);
+  void Connect(const char* address, unsigned int port4);
```

Comment:
Would've been nice to rename `port4` to just `port`, but it doesn't matter that much- Just thought I'd mention it on the off chance it does actually matter.